Dockerfile

```diff
@@ -1,16 +1,23 @@
-FROM alpine
+FROM golang:alpine3.13 AS builder
+LABEL stage=builder
+WORKDIR /workspace
+COPY . .
+RUN CGO_ENABLED=0 go build -o rest-server ./cmd/rest-server
+
+
+FROM alpine:3.13 AS final
+WORKDIR /app
 
 ENV DATA_DIRECTORY /data
 ENV PASSWORD_FILE /data/.htpasswd
+ENV PATH="/app:${PATH}"
 
 RUN apk add --no-cache --update apache2-utils
 
-COPY docker/create_user /usr/bin/
-COPY docker/delete_user /usr/bin/
-COPY docker/entrypoint.sh /entrypoint.sh
-COPY rest-server /usr/bin
+COPY docker/. .
```

Comment:
Is there a specific need to change the location where the helper scripts / entrypoint.sh are stored? I've been bitten a few times by container updates which "just" changed a few paths, but caused my derived containers to break.

Reply:
This is just my personal preference: Having all the additional stuff in the same location. But that is no "must". There also doesn't seem to be a best practice for that - so I can do it the way you want.